### ISSUE TITLE
MSBuild Targets Not Working on macOS

### ISSUE THREAD

**Issue by khalidabuhakmeh** [RainwayApp/bebop#64]
**Describe the bug**

The bebop tools don't seem to resolve properly on macOS so the compiler tools fail to compile the `.bop` files.

**To Reproduce**
1. Create a client library
2. Install bebop-tools from NuGet
3. Modify `.csproj` with "Getting Started" MSBuild target
4. Compile the application

**Expected behavior**

The `.bop` file is compiled into the corresponding `*.cs` file.

**Screenshots / Snippets**

Modifying the target directly and fixing the path separators allows me to compile.

From `Contracts.csproj`

```xml
    <ItemGroup>
        <Bebop Include="**/*.bop" OutputDir="./Models/" OutputFile="Records.g.cs" Namespace="Cowboy.Contracts" />
    </ItemGroup>

```

Modified MSBuild Target

```xml
	<Target Name="CompileBops" BeforeTargets="CoreCompile" DependsOnTargets="PrepareForBuild">

		<Exec
			Command="$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../tools/macos/bebopc')) --log-format MSBuild --cs &quot;$([System.IO.Path]::GetFullPath('%(Bebop.OutputDir)'))%(Bebop.OutputFile)&quot; --namespace %(Bebop.Namespace) --files $(_BebopSchemas)"
			EchoOff='true'
			StandardErrorImportance='high'
			StandardOutputImportance='low'
			ConsoleToMSBuild='true'
			ContinueOnError='false'
			StdOutEncoding='utf-8'>
			<Output TaskParameter="ConsoleOutput" PropertyName="_BebopCompiler" />
			<Output TaskParameter="ExitCode" PropertyName="MSBuildLastExitCode" />
		</Exec>

	</Target>
```

Here is the running sample I got working. To be clear, I had to download the NuGet package and copy the tools to the root folder. Not sure if I needed to do that but it seems to work.

It might make sense to migrate the `bebop-tools` over to a `dotnet global tool`.

**Here is a working sample https://github.com/khalidabuhakmeh/CowboyBebopSample**

**Bebop info:**
- Version: 2.0.2
- Runtime: .NET 5.0

**Desktop (please complete the following information):**
 - OS: macOS
 - Version: Big Sure

**Additional context**

**Comment by AndrewMD5** [comment]
Thanks for reporting this! I am working on a fix now.

**Comment by AndrewMD5** [comment]
This issue will be resolved in the 2.0.3 release later this evening. I'll close this issue then.

**Comment by AndrewMD5** [comment]
[Fixed in 2.0.3 on NuGet](https://www.nuget.org/packages/bebop-tools/2.0.3)
